run/logging-manual/README.md

```diff
@@ -42,7 +42,7 @@ test/deploy.sh
 
 ### runner.sh
 
-The `runner.sh` script is a helper that facilitates:
+The `runner.sh` script will:
```

Comment:
Saying bash files _will_ do something sounds a bit weird to me.

Maybe state what the program does rather than what it will do.

---

The `runner.sh` script:
- Deploys the service...
- Sets the ... variables...

Reply:
Ack